src/dappsExplorer/useDappFavoritedToast.tsx

```diff
@@ -0,0 +1,57 @@
+import React, { useEffect, useMemo, useState } from 'react'
+import { useTranslation } from 'react-i18next'
+import { SectionList } from 'react-native'
+import ToastWithCTA from 'src/components/ToastWithCTA'
+import { Dapp } from 'src/dapps/types'
+
+const TOAST_DISMISS_TIMEOUT = 5000
+
+const useDappFavoritedToast = (sectionListRef: React.RefObject<SectionList>) => {
```

Comment:
made this hook to look like the `useOpenDapp` hook, to decouple the favoriting functionality as much as possible from the screen itself